lib/node_runner_configuration.go

```diff
@@ -0,0 +1,46 @@
+//
+// NodeRunnerConfiguration has timeout features and transaction limit.
+// The NodeRunnerConfiguration is included in NodeRunnerStateManager and
+// these timeout features are used in ISAAC consensus.
+//
+package sebak
+
+import (
+	"time"
+
+	"boscoin.io/sebak/lib/common"
+)
+
+type NodeRunnerConfiguration struct {
+	TimeoutINIT       time.Duration
+	TimeoutSIGN       time.Duration
+	TimeoutACCEPT     time.Duration
+	TimeoutALLCONFIRM time.Duration
+
+	TransactionsLimit uint64
```

Comment:
@Charleslee522 Is This `TransactionsLimit` related to the #243? If yes and this PR is merged, I will apply this PR to #243.